delete-old-branches

```diff
@@ -86,6 +105,8 @@ main() {
             default_branch_protected "${br}" && echo "branch: ${br} is a default branch. Won't delete it" && continue
             branch_protected "${br}" && echo "branch: ${br} is likely protected. Won't delete it" && continue
             extra_branch_or_tag_protected "${br}" "branch" && echo "branch: ${br} is explicitly protected and won't be deleted" && continue
+            extra_branch_protected "${br}" && echo "branch: ${br} is explicitly protected and won't be deleted" && continue
```

Comment:
This has to be removed since this function has been replaced from `extra_branch_or_tag_protected`.